.ci/robot_framework/tests/tests_021_stress_ng.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
+Test Timeout    60 seconds
+
+Library    OperatingSystem
+Library    ../libs/TestUtils.py
+
+Resource   variables.robot
+
+*** Test Cases ***
+CPU 1 Stress Test
+    ${TEST_BOARD_IP}  Get Environment Variable                  TEST_BOARD_IP
+    ${stdout}=        SSH Command                               ${TEST_BOARD_IP}             .ci/scripts/stress-test.py --report --filter cpu_1_core
+    ${data}=          Evaluate                                  json.loads('''${stdout}[0]''')  json
+    Should Be True    ${data['actual']} <= ${data['expected']}
+
+CPU 4 Stress Test
+    ${TEST_BOARD_IP}  Get Environment Variable                  TEST_BOARD_IP
+    ${stdout}=        SSH Command                               ${TEST_BOARD_IP}             .ci/scripts/stress-test.py --report --filter cpu_4_core
+    ${data}=          Evaluate                                  json.loads('''${stdout}[0]''')  json
+    Should Be True    ${data['actual']} <= ${data['expected']}
```

Comment:
should not be  `>=`

```suggestion
    Should Be True    ${data['actual']} >= ${data['expected']}
```
I mean, the expectation should be the bare minimum achievable mark.